src/quokka_editor_back/conftest.py

```diff
@@ -43,6 +44,9 @@ async def initialize_tests(request):
 async def client() -> TestClient:
     return TestClient(app=asgi_app)
 
+@pytest.fixture
+def websocket():
+    return AsyncMock(spec=WebSocket)
```

Comment:
nice usage of spec. I learned it few days ago and it's a really nice feature. :D